gnome-base/dconf-editor/dconf-editor-43.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+VALA_MIN_API_VERSION="0.40"
+
+inherit gnome.org gnome2-utils meson vala xdg
+
+DESCRIPTION="Graphical tool for editing the dconf configuration database"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/dconf-editor"
+
+LICENSE="GPL-3+"
+SLOT="0"
+
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86 ~x86-linux"
+
+RDEPEND="
+	>=gnome-base/dconf-0.26.1
+	>=dev-libs/glib-2.55.1:2
+	>=x11-libs/gtk+-3.22.27:3
+	>=gui-libs/libhandy-1.6.0:1
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	$(vala_depend)
+	dev-libs/libxml2:2
+	>=sys-devel/gettext-0.19.8
+	virtual/pkgconfig
+"
+PATCHES=(
+	"${FILESDIR}/meson-0.50.patch"
```

Comment:
I assume this is to silence this warning?

````
WARNING: Project specifies a minimum meson_version '>= 0.41.0' but uses features which were added in newer versions:
 * 0.50.0: {'install arg in configure_file'}
````

I wouldn't bother with patching it in Gentoo, but if you want to, if you can submit a patch upstream to use Meson's `gnome.post_install` function, which requires Meson 0.57. See https://gitlab.gnome.org/GNOME/five-or-more/-/merge_requests/25 for an example of doing this.